specification/edgeorderpartner/resource-manager/Microsoft.EdgeOrderPartner/preview/2020-12-01-preview/edgeorderpartner.json

```diff
@@ -0,0 +1,905 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ListOperationsPartner": {
+            "$ref": "./examples/ListOperationsPartner.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrderPartner/locations/{location}/productFamilies/{familyIdentifier}/inventories/{serialNumber}/manageInventoryMetadata": {
+      "post": {
+        "tags": [
+          "PartnerInventory"
+        ],
+        "summary": "API for updating inventory metadata and inventory configuration",
+        "operationId": "ManageInventoryMetadata",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/familyIdentifierParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/locationParameter"
+          },
+          {
+            "$ref": "#/parameters/serialNumberParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "manageInventoryMetadataRequest",
+            "in": "body",
+            "description": "Updates inventory metadata and inventory configuration",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManageInventoryMetadataRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "\"Success\""
+          },
+          "202": {
+            "description": "\"Request is accepted to manage inventory metadata\""
+          },
+          "204": {
+            "description": "\"No content\""
+          },
+          "default": {
+            "description": "Error response describing reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "ManageInventoryMetadata": {
+            "$ref": "./examples/ManageInventoryMetadata.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EdgeOrderPartner/locations/{location}/productFamilies/{familyIdentifier}/inventories/{serialNumber}/manageLink": {
+      "post": {
+        "tags": [
+          "PartnerInventory"
+        ],
+        "summary": "API for linking management resource with inventory",
+        "operationId": "ManageLink",
```

Comment:
Usually it is recommended to write it as `<Group>_<Operation>`, especially if there is pending more operations to be added. You can refer to json from any major services.